state/src/item/metadata.rs

```diff
@@ -34,6 +34,7 @@ pub struct Metadata {
     term: TermMetadata,
     seq: u64,
     params: Option<CommonParams>,
+    term_params: Option<CommonParams>,
```

Comment:
What's the difference between `params` and `term_params`?

Reply:
`term_params` is the common parameter at the start of the term.
Some parts in our implementation (e.g. tendermint) depend on this value.